src/content/quickstart/openshift/_index.md

```diff
@@ -0,0 +1,135 @@
+---
+date: 2020-02-21T13:36:18+01:00
+title: "OpenShift (AWS)"
+---
+
+## AWS
+
+### openshift-install and pull-secret
+
+Download the **openshift-install** and _oc_ tools, and copy your _pull secret_ from:
+
+> https://cloud.redhat.com/openshift/install/aws/installer-provisioned
+
+
+### Make sure you're logged in aws cli
+
+```bash
+$ aws configure
+AWS Access Key ID [None]: AKIAIOSFODNN7EXAMPLE
+AWS Secret Access Key [None]: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+Default region name [None]: us-east-2
+Default output format [None]: text
+```
+
+
+### Create and deploy cluster A
+
+In this step you will deploy cluster A, with the default IP CIDRs
+
+| Pod CIDR     | Service CIDR |
+|--------------|--------------|
+|10.128.0.0/14 |172.30.0.0/16 |
+
+
+```bash
+openshift-install create install-config --dir cluster-a
+```
+
+```bash
+openshift-install create cluster --dir cluster-a
+```
+
+The create cluster step will take some time, you can create Cluster B in parallel if you wish.
+
+### Create and deploy cluster B
+
+In this step you will deploy cluster B, modifying the default IP CIDRs
+
+| Pod CIDR     | Service CIDR |
+|--------------|--------------|
+|10.132.0.0/14 |172.31.0.0/16 |
+
+
+```bash
+openshift-install create install-config --dir cluster-b
+```
+
+
+Change the POD IP network, please note it’s a /14 range by default so you need to use 
++4 increments for “128”, for example: 10.132.0.0, 10.136.0.0, 10.140.0.0, ...
+ 
+```bash
+sed -i 's/10.128.0.0/10.132.0.0/g' cluster-b/install-config.yaml
+```
+
+Change the service IP network, this is a /16 range by default, so just use +1 increments
+for “30”: for example: 172.31.0.0, 172.32.0.0, 172.33.0.0, ...
+
+```bash
+sed -i 's/172.30.0.0/172.31.0.0/g' cluster-b/install-config.yaml
+```
+
+
+And finally deploy
+
+```bash
+openshift-install create cluster --dir cluster-b
+```
+
+### Make your clusters ready for submariner
+
+Submariner gateway nodes need to be able to accept traffic over ports 4500/UDP and 500/UDP
+when using IPSEC. In addition we use port 4800/UDP to encapsulate traffic from the worker nodes
+to the gateway nodes and ensuring that Pod IP addresses are preserved.
+
+Additionally, the default Openshift deployments don't allow assigning an elastic public IP
+to existing worker nodes, something that it's necessary at least on one end of the IPSEC connections. 
+
+To handle all those details we provide a script that will prepare your AWS OpenShift deployment
+for submariner, and will create an additional gateway node with an external IP.
+
+```bash
+
+curl https://raw.githubusercontent.com/submariner-io/submariner/master/tools/openshift/ocp-ipi-aws/prep_for_subm.sh -L -O
+chmod a+x ./prep_for_subm.sh
+
+./prep_for_subm.sh cluster-a     # respond yes when terraform asks
+./prep_for_subm.sh cluster-b      # respond yes when terraform asks
```

Comment:
I think it is only required to run this script against one of the public clusters, as it mainly opens ports for the Broker.
(note the opened bug, where these ports 4500 & 500, are hard coded in the script)
Creating the gateway labels will be done in another step (or with subctl), won't it ?

Reply:
No, it needs to be run on all, since it will also open the port 4800 between worker instances.

The gateway labels are automatically assigned, or otherwise asked by subctl at runtime.